src/dashboard/src/components/helpers.py

```diff
@@ -204,6 +207,35 @@ def get_server_config_value(field):
     except:
         return ''
 
+def get_atom_levels_of_description(clear=True):
+    url = get_setting('dip_upload_atom_url')
+    if not url:
+        raise Exception("AtoM URL not defined!")
```

Comment:
Would be nice if this raised a more specific exception, so we don't silence other things catching it.  Similarly, do we want `fetch_levels_of_description_from_atom` to catch all Exceptions, or just the AtoM-related ones we're raising here?